src/excmds.ts

```diff
@@ -5899,6 +5899,43 @@ export async function issue() {
     textarea.value = template
 }
 
+/**
+ * generates qr code for the given text
+ */
+//#content
+export async function text2qr(...args: string[]) {
+    let text: string = null
+    let isParsed = false
+    let openMode = null
+    while (!isParsed) {
+        switch (args[0]) {
+            case "--window":
+                openMode = winopen
+                args.shift()
+                break
+            case "--current":
+                openMode = open
+                args.shift()
+                break
+            default:
+                isParsed = true
+                break
+        }
```

Comment:
I'd like a timeout argument, e.g. `--timeout=5`, after which the page would be closed automatically. I am imagining people using the command with their phones already out ready to scan, so it would save them from having to close the page again afterwards

Reply:
Should i enable the timeout by default?  If yes then i think a default timeout of 5secs should be enough.

Reply:
I think binding to yq by default to `text2qr --timeout=5` rather than setting it internally would be a good idea, since that way people can see how to change it easily by looking at the bind

Reply:
addressed this below. Pls check commit #[cb29ce4](https://github.com/tridactyl/tridactyl/pull/4728/commits/cb29ce41a5878178e0f46fbfc530938841f94e64)